### PR TITLE
Feature/waf policy groups

### PR DESCRIPTION
**Pull request by sandromodarelli**
**Did you labeled this PR?**

YES

**What this PR does / why we need it**:
waf policy groups and policies handling

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
waf policy groups and policies handling
```

### CODE REVIEW

File: pystackpath/stacks/wafsites/__init__.py [hunk 1]
```diff
@@ -9,3 +10,6 @@ def rules(self):
 
     def ddos(self):
         return Ddos(self._client, f"{self._base_api}/sites/{self.id}")
+
+    def policy_groups(self):
+        return PolicyGroups(self._client, f"{self._base_api}/sites/{self.id}")
```
Comment (prometherion) [inline pystackpath/stacks/wafsites/__init__.py]: Missing return at EOF

File: pystackpath/stacks/wafsites/policy_groups/__init__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        response = self._client.post(f"{self._base_api}/policy_groups/{self.id}/enable")
+
+    def disable(self):
+        """
+        Disable all policies in a WAF policy group
+        """
+        response = self._client.post(f"{self._base_api}/policy_groups/{self.id}/disable")
+
+    def policies(self):
+        return Policies(self._client, f"{self._base_api}/policy_groups/{self.id}")
```
Comment (prometherion) [inline pystackpath/stacks/wafsites/policy_groups/__init__.py]: Missing return at EOF

File: pystackpath/stacks/wafsites/__init__.py (hunk 1 above)
Comment (sandromodarelli) [reply pystackpath/stacks/wafsites/__init__.py]: fixed

Comment (prometherion) [reply pystackpath/stacks/wafsites/__init__.py]: @sandromodarelli here

**Review by prometherion — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]

**Pull request merged**